frontend/app/module/election/page/ElectionStatusPage.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { beforeEach, describe, expect, test } from "vitest";
+
+import { ElectionStatusPage } from "app/module/election";
+import { overrideOnce, render, screen } from "app/test/unit";
+
+import { ElectionProvider, ElectionStatusProvider } from "@kiesraad/api";
+import { electionDetailsMockResponse } from "@kiesraad/api-mocks";
+
+const renderElectionStatusPage = () =>
+  render(
+    <ElectionProvider electionId={1}>
+      <ElectionStatusProvider electionId={1}>
+        <ElectionStatusPage />
+      </ElectionStatusProvider>
+    </ElectionProvider>,
+  );
+
+describe("DataEntryHomePage", () => {
```

Comment:
```suggestion
describe("ElectionStatusPage", () => {
```

Reply:
Fixed in 623a7a1ad6c1fea13ddaba577f23e2f61d29b1a4